src/database/seeders/20191001085625-accommodations.js

```diff
@@ -0,0 +1,48 @@
+/* eslint-disable no-unused-vars */
+/**
+ * @func up
+ * @param {*} queryInterface
+ * @param {*} Sequelize
+ * @return {func} bulkInsert
+ */
+export function up(queryInterface, Sequelize) {
+  return queryInterface.bulkInsert('Accomodations', [{
+    accommodation_name: 'Kigali Marriott Hotel',
+    location: 1,
+    createdAt: new Date(),
+    updatedAt: new Date()
+  },
+  {
+    accommodation_name: 'Kigali Serena Hotel',
+    location: 1,
+    createdAt: new Date(),
+    updatedAt: new Date()
+  },
+  {
+    accommodation_name: 'Kigali Radisson Blu Hotel',
+    location: 1,
+    createdAt: new Date(),
+    updatedAt: new Date()
+  },
+  {
+    accommodation_name: 'Four Seasons Hotel',
+    location: 2,
+    createdAt: new Date(),
+    updatedAt: new Date()
+  },
+  {
+    accommodation_name: 'EKO Hotels & Suites',
+    location: 3,
+    createdAt: new Date(),
+    updatedAt: new Date()
+  }], {});
+}
+/**
+ * @func down
+ * @param {*} queryInterface
+ * @param {*} Sequelize
+ * @returns {func} bulkDelete
+ */
+export function down(queryInterface, Sequelize) {
+  return queryInterface.bulkDelete('Accomodations', null, {});
+}
```

Comment:
Just seen on the database room type is still there under accommodations whereas I thought we will only have the total number of rooms available there?
I understand it might not have been your task but if the change doesn't break a lot of the code you can do it since it's small

Reply:
Yes I left it there so that it doesn't crash @TgeeT endpoint, right now he is fixing it he will also make sure he removes it @didasmbalanya